RealmTasks Apple/RealmTasks iOS/ViewController.swift

```diff
@@ -135,15 +157,14 @@ final class ViewController<Item: Object, Parent: Object where Item: CellPresenta
                 return
             }
         } else {
-            var row: Int = 0
-            try! items.realm?.write {
-                row = items.filter("completed = false").count
-                items.insert(Item(), atIndex: row)
-            }
+            items.realm?.beginWrite()
```

Comment:
Sorry, GitHub's UI fooled me into thinking this comment was in reference to a different place where I use `beginWrite()`. This could use `uiWriteNoUpdateList()`, but that would turn the `let cell:` declaration above into a `var cell:`, but this could certainly use a `finishUIWrite()`, so I'll change that.